### ISSUE TITLE
Crash when editing new EventPage

### ISSUE THREAD

**Issue by rjsparks** [ietf-tools/www#278]
```Internal Server Error: /how/meetings/

FieldDoesNotExist at /how/meetings/
EventPage has no field named 'pk'

Request Method: GET
Request URL: http://www.ietf.org/how/meetings/
Django Version: 3.2.13
Python Executable: /usr/local/bin/python
Python Version: 3.10.9
Python Path: ['/app', '/usr/local/bin', '/usr/local/lib/python310.zip', '/usr/local/lib/python3.10', '/usr/local/lib/python3.10/lib-dynload', '/usr/local/lib/python3.10/site-packages']
Server time: Thu, 30 Mar 2023 08:37:38 +0100
Installed Applications:
('ietf.home',
 'ietf.blog',
 'ietf.events',
 'ietf.search',
 'ietf.forms',
 'ietf.snippets',
 'ietf.standard',
 'ietf.topics',
 'ietf.glossary',
 'ietf.utils',
 'ietf.bibliography',
 'ietf.images',
 'ietf.documents',
 'ietf.iesg_statement',
 'wagtail.contrib.forms',
 'wagtail.contrib.redirects',
 'wagtail.embeds',
 'wagtail.sites',
 'wagtail.users',
 'wagtail.snippets',
 'wagtail.documents',
 'wagtail.images',
 'wagtail.search',
 'wagtail.contrib.search_promotions',
 'wagtail.admin',
 'wagtail',
 'wagtail.contrib.settings',
 'wagtail.contrib.table_block',
 'wagtail.contrib.routable_page',
 'wagtail.contrib.modeladmin',
 'modelcluster',
 'compressor',
 'taggit',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.admindocs',
 'django.contrib.sitemaps',
 'mod_wsgi.server',
 'analytical',
 'wagtailmarkdown',
 'wagtailorderable')
Installed Middleware:
['django.middleware.cache.UpdateCacheMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'wagtail.contrib.redirects.middleware.RedirectMiddleware',
 'django.middleware.cache.FetchFromCacheMiddleware']


Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 829, in _resolve_lookup
    current = current[bit]

During handling of the above exception ('EventListingPage' object is not subscriptable), another exception occurred:
  File "/usr/local/lib/python3.10/site-packages/django/db/models/options.py", line 608, in get_field
    return self.fields_map[field_name]

During handling of the above exception ('pk'), another exception occurred:
  File "/usr/local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.10/site-packages/wagtail/models/__init__.py", line 622, in _get_response
    response = response.render()
  File "/usr/local/lib/python3.10/site-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python3.10/site-packages/django/template/response.py", line 83, in rendered_content
    return template.render(context, self._request)
  File "/usr/local/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 170, in render
    return self._render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 162, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 162, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/defaulttags.py", line 404, in render
    return strip_spaces_between_tags(self.nodelist.render(context).strip())
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/defaulttags.py", line 308, in render
    match = condition.eval(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/defaulttags.py", line 892, in eval
    return self.value.resolve(context, ignore_failures=True)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 671, in resolve
    obj = self.var.resolve(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 796, in resolve
    value = self._resolve_lookup(context)
  File "/usr/local/lib/python3.10/site-packages/django/template/base.py", line 837, in _resolve_lookup
    current = getattr(current, bit)
  File "/app/ietf/events/models.py", line 204, in upcoming_events
    pk__in=self.promoted_events.all().values_list(
  File "/usr/local/lib/python3.10/site-packages/modelcluster/queryset.py", line 506, in values_list
    get_model_field(self.model, f)
  File "/usr/local/lib/python3.10/site-packages/modelcluster/utils.py", line 69, in get_model_field
    field = subject_model._meta.get_field(field_name)
  File "/usr/local/lib/python3.10/site-packages/django/db/models/options.py", line 610, in get_field
    raise FieldDoesNotExist("%s has no field named '%s'" % (self.object_name, field_name))

Exception Type: FieldDoesNotExist at /how/meetings/
Exception Value: EventPage has no field named 'pk'
```

**Comment by mgax** [comment]
This was likely fixed in #370.

**Comment by rjsparks** [comment]
closing this based on above comment
